Formula/l/llamafile.rb

```diff
@@ -0,0 +1,43 @@
+class Llamafile < Formula
+  desc "Distribute and run LLMs with a single file"
+  homepage "https://github.com/Mozilla-Ocho/llamafile"
+  url "https://github.com/Mozilla-Ocho/llamafile/archive/refs/tags/0.6.tar.gz"
+  sha256 "9ac8449268287c45fb015f7068b223be7336c6d7f8170f3adb9a370b65e8b9f0"
+  license "Apache-2.0"
+  head "https://github.com/Mozilla-Ocho/llamafile.git", branch: "main"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "make"
+
+  skip_clean "bin/llamafile",
+             "bin/llamafile-convert",
+             "bin/llamafile-perplexity",
+             "bin/llamafile-quantize",
+             "bin/llava-quantize",
+             "bin/zipalign"
+
+  def install
+    # Call `make` as `gmake` to use GNU `make`
+    system "gmake", "-j8"
+    system "gmake", "install", "PREFIX=#{prefix}"
+
+    %w[
+      llamafile
+      llamafile-convert
+      llamafile-perplexity
+      llamafile-quantize
+      llava-quantize
+      zipalign
+    ].each do |bin_file|
+      chmod "+x", "#{bin}/#{bin_file}"
+    end
```

Comment:
Doesn't `make` do this?

Reply:
It does not. Would filing an issue upstream and adding a comment to track that in the formula work?

Reply:
We should just wait for a release that fixes it then

Reply:
Dropped this since it's fixed in my source build of 0.6.1.